File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/java/F19.P1.java
class Main {
    static int concurrentSum() {
        // BEGIN PROBE F19.P1
        var first = java.util.concurrent.CompletableFuture.supplyAsync(() -> 20);
        var second = java.util.concurrent.CompletableFuture.supplyAsync(() -> 22);
        int sum = first.join() + second.join();
        return sum;
        // END PROBE F19.P1
    }

    public static void main(String[] args) {
        System.out.println(concurrentSum());
    }
}

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/kotlin/F19.P1.kt
fun probe(): Int {
    // BEGIN PROBE F19.P1
    val f1 = java.util.concurrent.CompletableFuture.supplyAsync { 20 }
    val f2 = java.util.concurrent.CompletableFuture.supplyAsync { 22 }
    val sum = f1.join() + f2.join()
    // END PROBE F19.P1
    return sum
}

fun main() {
    println(probe())
}
